marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -1,7 +1,8 @@
+-- editorconfig-checker-disable-file
+
```

Comment:
can remove

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.